DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,11 +164,15 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
```

Comment:
@Dr15Jones 
what do you think is missing in this file so that it will be seen in 1102, which is just a read of a file and then write with 
"fastCloning = False"
?

I was able to reproduce by merging this PR in 10934
(in my pre-signoff tests I missed to run the matrix locally)